CRM/Civixero/Invoice.php

```diff
@@ -594,7 +594,14 @@ protected function pushToXero($accountsInvoice, $connector_id) {
     }
     try {
       /** @noinspection PhpUndefinedMethodInspection */
-      return $this->getSingleton($connector_id)->Invoices($accountsInvoice);
+      $result = $this->getSingleton($connector_id)->Invoices($accountsInvoice);
+      // Note: One day it would be nice to switch to the new Xero PHP SDK method for pushing invoices.
+      // For some reason pushing certain invoices seems to result in a completely empty response
```

Comment:
@mattwire on my last round of work on this I found the completely empty responses had headers that indicated the issue (throttling in the case I was looking at)

I did some stuff to look in the headers but I don't think it affected all paths

Reply:
Ah, interesting. We can leave this open until I get chance to look at that then :-)

Reply:
Ah - this is where I did it https://github.com/eileenmcnaughton/nz.co.fuzion.civixero/commit/48be1f9c714bd5c2a934fb60d0e85301465a0ab5#diff-95bea40777ff719d8f405079e553aeef371b093a33d0970b08b134694265402eR311-R326